frontend/www/js/omegaup/components/course/Mine.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="card">
+    <div class="card-header">
+      <h3 class="card-title">{{ T.courseListAdminCourses }}</h3>
+    </div>
+
+    <div class="m-3" v-if="isMainUserIdentity">
+      <div class="float-right">
+        <a class="btn btn-primary" href="/course/new/">{{ T.courseNew }}</a>
+      </div>
+      <h1>&nbsp;</h1>
+    </div>
+    <template v-if="courses.admin.activeTab !== ''">
+      <omegaup-course-filtered-list
+        v-bind:courses="courses.admin"
+        v-bind:activeTab="courses.admin.activeTab"
+      ></omegaup-course-filtered-list>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import T from '../../lang';
+import * as UI from '../../ui';
```

Comment:
nit:

```suggestion
import * as ui from '../../ui';
```

hemos estado intentando mover esto a minúsculas en todos los demás lugares.

Reply:
https://github.com/omegaup/omegaup/pull/4347 debería terminar esa migración.